internal/plugincommon/validations.go

```diff
@@ -15,3 +16,40 @@ func ValidateFChain(fChain map[cciptypes.ChainSelector]int) error {
 	}
 	return nil
 }
+
+var ErrValidationError = errors.New("report validation errors")
+var ErrInvalidReport = errors.New("invalid report")
+
+// NewErrValidatingReport is returned when the report could not be validated due to an error.
+func NewErrValidatingReport(err error) error {
+	return &errWrappedValidatingReport{err: err}
+}
```

Comment:
Do we need a separate struct? I thought `fmt.Errorf` works just fine with `errors.Is`?

Reply:
`errors.Is` only works with error constants, you need the struct so that `Unwrap` returns a constant. I think the alternative would be to return `fmt.Errorf("error reason: %w", ErrValidatingReport)`.